### DISCUSSION TITLE
Dashboards.

### DISCUSSION THREAD

**Discussion by mike667** [highlight/highlight#6880] [Q&A]
Hello, I'm new one here. I want to use highlight io with my react app.

I saw in this [article](https://www.highlight.io/blog/frontend-observability) I can have dashboard. But I cannot find it in my project.
In article looks like dashboard page, with aggregated metrics by users and time. But in my project I have only sessions, errors and logs pages. I can find metrics in each user session, but I cannot find aggregated metrics graphic.

Could you help me? Maybe I have to upgrade my plan.

I tried to find something in the [docs](https://www.highlight.io/docs/getting-started/client-sdk/reactjs), but they aren't user friendly much.

Thank you.

**Comment by Vadman97** [comment]
Hi Mike! 

The dashboards / analytics pages are something we are reworking in the coming quarter. You can find the existing functionality at the direct links: https://app.highlight.io/analytics for the home dashboard, and https://app.highlight.io/dashboards for custom ones. Some functionality may be incomplete though, and we will be revamping this significantly in the next two months. Stay tuned for more, and follow the following issue if you are interested: https://github.com/highlight/highlight/issues/5915
